exodus/reports/templates/report_details.html

```diff
@@ -67,6 +66,9 @@ <h3>
     </div>
     <div class="col-md-8 col-12 text-right mt-2">
       <a class="link main-link" target="_blank" rel="noreferrer" href="https://play.google.com/store/apps/details?id={{ report.application.handle }}">{% trans "See on Google Play" %}</a>
+      {% if request.user.is_superuser %}
+        <br><a class="link main-link" href="{% url 'admin:reports_report_delete' report.id %}">{% trans "Delete this report" %}</a>
```

Comment:
The translation of "Delete this report" is not done, is it on purpose?

Reply:
The whole translation will be done in another branch.